app/Services/TradeManager.php

```diff
@@ -265,7 +395,7 @@ public function confirmTrade($data, $user) {
             }
 
             if ($trade->is_sender_trade_confirmed && $trade->is_recipient_trade_confirmed) {
-                if (!(Settings::get('open_transfers_queue') && (isset($trade->data['sender']['characters']) || isset($trade->data['recipient']['characters'])))) {
+                if (!Settings::get('open_trades_queue') || !(Settings::get('open_transfers_queue') && (isset($trade->data['sender']['characters']) || isset($trade->data['recipient']['characters'])))) {
```

Comment:
(just using the suggestion thing for ease of demonstration)
```suggestion
                if ((!Settings::get('open_trades_queue') || !(Settings::get('open_transfers_queue')) && (isset($trade->data['sender']['characters']) || isset($trade->data['recipient']['characters'])))) {
```

Reply:
Thought to double-check the logic here, realized that "or" should probably be its own little sub-clause?